packages/drivers/detox/index.ts

```diff
@@ -0,0 +1,692 @@
+import {
+    TestingFrameworkAPICatalog,
+    TestingFrameworkDriver,
+} from "@wix-pilot/core";
+import detox from "detox";
+import { expect } from '@jest/globals';
+import * as fs from "fs";
+import * as path from 'path';
+
+export class DetoxFrameworkDriver implements TestingFrameworkDriver {
+    async captureViewHierarchyString(): Promise<string> {
+        try {
+            return await detox.device.generateViewHierarchyXml(true);
+        } catch (_error) {
+            return "NO INNER VIEW HIERARCHY FOUND, APP IS PROBABLY NOT LAUNCHED YET";
```

Comment:
hmm let's try to improve this message
```suggestion
            return "NO ACTIVE APP FOUND, LAUNCH THE APP TO SEE THE VIEW HIERARCHY";
```